cypress/e2e/page/index-pagetests.cy.js

```diff
@@ -51,26 +51,31 @@ describe("tests on the index page via store interaction", () => {
                 cy.assertNextPageAccess("categorization", true);
             });
 
-            // Description: User selects data dictionary alone
+            // Description: User cannot select a data dictionary alone
             // Expected results: Categorization nav and next page button remain disabled
-            it("select data dictionary only", () => {
+            it("Selecting a data dictionary without a data table is not possible", () => {
 
                 // 1. Assert that categorization nav item and next button are disabled
                 cy.assertNextPageAccess("categorization", false);
 
-                // 2. Select participants dictionary
+                // 2. Check that data dictionary file select button is disabled
+                cy.get("[data-cy='data-dictionary-selector']")
+                    .find("label")
+                    .first()
+                    .should("have.class", "disabled-file-selector-button");
+
+                // 3. Attempt to click on 'Choose file' button for data dictionary
                 cy.get("[data-cy='data-dictionary-selector']")
                     .contains("Choose file")
-                    .click()
-                    .selectFile(dataFolder + p_dataset.data_dictionary);
+                    .click();
```

Comment:
If I understand correctly, this click should not be possible at this stage, right? Is there any way to assert that directly? If not, I'd add a brief explanation why to the comment above - because I was expecting something like `.should("be.disabled")` like we're usually using.

Reply:
The `click` itself is possible (producing no response), but the `selectFile` makes the test fail because label that would be clicked on in `file-selector` has been disabled. There is a way to check for a Cypress error (e.g. 'is not clickable'), but apparently [this is highly discouraged](https://github.com/cypress-io/cypress/discussions/21150).

>If not, I'd add a brief explanation why to the comment above - because I was expecting something like .should("be.disabled") like we're usually using.

👍

Reply:
>Oh, forgot an important point: now that the e2e tests can (at least in principle) pass again, should we exclude the annotation page e2e tests until they are reasonably passing? The goal would be to have from now and going forward fully green tests again!

Yes, the annotation page e2e tests (and really the download page e2e tests too - even though it is sort of passing now) should be fully functional by the end of Phase 2 refactor work.

Reply:
Just to be clear about the part with the annotation-test: If we know they won't pass until phase 2 is done, maybe let's disable them for now so the e2e tests can also pass now, and seeing them fail conveys an actual problem as we're finishing phase 2.